ui-automation-tests/support/steps/shared-steps.ts

```diff
@@ -161,3 +161,8 @@ Then(/^there should be no accessibility violations$/, async function (this: Test
     const results = await new AxePuppeteer(this.page).withTags(["wcag21aa", "wcag22aa"]).analyze();
     assert.equal(results.violations.length, 0, "Accessibility Violations Detected : " + JSON.stringify(results.violations));
 });
+
+Then(/^they should see the link '(.*)'$/, async function (text: string) {
```

Comment:
This code is not needed, because the check in the feature that uses it is not needed as well, as per my previous comment